### ISSUE TITLE
Big scripts cause lag and crashes in lua

### ISSUE THREAD

**Issue by xLevitate** [BalazsJako/ImGuiColorTextEdit#136]
In the Lua text editor, big scripts will lag and crash, I was wondering if there is any way to fix this.

**Comment by RP1Nate** [comment]
Did you ever figure this out?

**Comment by griffi-gh** [comment]
yeah it segfaults, seems to be caused by stack overflow in some random c++ std regex function.
using `where` in gdb prints tens of thouthands of lines.

**Comment by sten-code** [comment]
I had this exact issue, I replaced all regex things with the boost library equivalent which fixed this issue
